ui/browser/views/navbar/location.jsx

```diff
@@ -158,11 +158,12 @@ class Location extends Component {
           </span>
         </div>
         <input id="urlbar-input"
-          className={INPUT_BAR_STYLE}
-          hidden={!showURLBar}
+          is="tofino-input"
```

Comment:
It turns out using is="" here isn't necessary, and the fix is due to the change from value to defaultValue